connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -91,13 +94,15 @@
     private static final Logger log = LoggerFactory.getLogger(DistributedHerder.class);
 
     private static final long RECONFIGURE_CONNECTOR_TASKS_BACKOFF_MS = 250;
+    private static final int BULK_THREAD_POOL_SIZE = 8;
 
     private final Time time;
 
     private final int workerSyncTimeoutMs;
     private final int workerUnsyncBackoffMs;
 
     private final ExecutorService forwardRequestExecutor;
+    private final ExecutorService bulkExecutor;
```

Comment:
nitpick: a more descriptive name would be helpful, but I'm struggling to think of one. Maybe `startupShutdownExecutor`?